lib/location/Kconfig

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig LOCATION
+	bool "Location"
+	depends on NRF_MODEM_LIB
+	help
+	  TODO
+
+if LOCATION
+
+config LOCATION_METHOD_GNSS
+	bool "Allow GNSS to be used for obtaining the location"
+	default y
+
+config LOCATION_METHOD_CELLULAR
+	bool "Allow cellular positioning to be used for obtaining the location"
+	select MULTICELL_LOCATION
+	default y
+
+config LOCATION_METHOD_WIFI
+	bool "Allow WiFi positioning to be used for obtaining the location"
+	depends on WIFI
+	select REST_CLIENT
+	select CJSON_LIB
+	select NET_MGMT
+	select NET_MGMT_EVENT
+	select NET_MGMT_EVENT_INFO
+	select NET_L2_WIFI_MGMT
+	default y if WIFI
+
+if LOCATION_METHOD_GNSS
+config LOCATION_METHOD_GNSS_AGPS_EXTERNAL
+	bool "Requesting A-GPS data is handled by the application outside of Location library."
+	depends on NRF_CLOUD_AGPS
+	default n
+	help
+	  Allow application to download the A-GPS data and merely feed it to Location library for
+	  processing. If enabled, Location library throws event LOC_EVT_GNSS_ASSISTANCE_REQUEST when
+	  assistance is needed.
+endif # LOCATION_METHOD_GNSS
+
+if LOCATION_METHOD_WIFI
+
+config LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD
+	bool "nRF Cloud location service"
+	default y
+	depends on NRF_CLOUD
+	help
+	  Use nRF Cloud location service.
+
+config LOCATION_METHOD_WIFI_SERVICE_HERE
+	bool "WiFi positioning using Here service"
+	default n
+	help
+	  Use HERE location service.
+
+config LOCATION_METHOD_WIFI_SERVICE_SKYHOOK
+	bool "WiFi positioning using Skyhook service"
+	default n
+	depends on MODEM_JWT
+	help
+	  Use Skyhook location service.
+
+config LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT
+	int "Max number of WiFi scanning results"
+	default 10
+	help
+	  Maximum number of WiFi scanning results to use when creating HTTP request.
+	  Increasing the max number will increase the library's RAM usage.
+
+config LOCATION_METHOD_WIFI_REST_RECV_BUF_SIZE
+	int "Receive buffer size"
+	default 1024
+	help
+	  Size of the buffer used to store the response from the WiFi positioning
+	  service.
+
+if LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD
+
+config LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD_HOSTNAME
+	string "nRF Cloud REST API hostname"
+	default NRF_CLOUD_REST_HOST_NAME
```

Comment:
A question here: we are removing `LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD_SEC_TAG`. Should we remove this too? I do see some potential use for it during development but is that enough to justify this config?
@jhirsi @tokangas

Reply:
removed